tools/utils/categories.ts

```diff
@@ -25,12 +25,8 @@ for (const categoryId of categoryIds) {
 
   for (const rule of rules) {
     const ruleCategories = rule.meta.docs?.categories
-    // Throw if rule does not have a category
-    if (!ruleCategories?.length) {
-      throw new Error(`Rule "${rule.ruleId}" does not have any category.`)
-    }
 
-    if (ruleCategories.includes(categoryId) && rule.meta.docs?.excludeFromConfig !== true) {
+    if (ruleCategories?.includes(categoryId)) {
```

Comment:
@yannbf Do you think this is fine?

Reply:
I don't think so, the excludeFromConfig was added for rules which could be tested internally but shouldn't be exposed in the docs so it makes sense to keep testing that

Reply:
Ah, I thought `excludeFromConfig` was the replacement flag for "0 categories" rules. Should I just keep the `ruleCategories?.includes` bit then?

Reply:
sure!

Reply:
Hold on, I was watching this on mobile and thought I had added something I should remove, and I confused myself. Now I see the diff, I think my last rebase accidentally rerere'd this code. In my first rebase, I recall I explicitly kept it.

I will re-add `&& rule.meta.docs?.excludeFromConfig !== true`.

I added `excludeFromConfig` on line 25 of [`lib/rules/meta-satisfies-type.ts`](https://github.com/storybookjs/eslint-plugin-storybook/pull/200/files#diff-bdf419d5599fe66e3a738846abf03a79f276e0c1ed2fa0b047e285c74526509fR25), because I thought it had the same meaning. So this is where I may still be wrong. Was that correct to do?

Reply:
Reinstated.

Reply:
Thanks, doing some tests and will merge afterward!